From linux-usb-owner@vger.kernel.org Mon Feb 11 04:17:58 2008
From: Peter Korsgaard <jacmet@sunsite.dk>
Date: Mon, 11 Feb 2008 13:16:26 +0100
Subject: USB: ehci-fsl: mpc834x config symbol is PPC_MPC834x, not MPC834x
To: linux-usb@vger.kernel.org, david-b@pacbell.net
Cc: Peter Korsgaard <jacmet@sunsite.dk>
Message-ID: <1202732186-7345-1-git-send-email-jacmet@sunsite.dk>


The config symbol for mpc834x processors is CONFIG_PPC_MPC834x,
not CONFIG_MPC834x.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -72,7 +72,7 @@ config USB_EHCI_FSL
 	bool
 	depends on USB_EHCI_HCD
 	select USB_EHCI_ROOT_HUB_TT
-	default y if MPC834x || PPC_MPC831x
+	default y if PPC_MPC834x || PPC_MPC831x
 	---help---
 	  Variation of ARC USB block used in some Freescale chips.
 
